kernel/will-it-scale.py.data/will-it-scale.yaml

```diff
@@ -0,0 +1,188 @@
+# To run all tests use following stanza instead
+#testcase: !mux
+#    All:
+#        name: All
+#        postprocess: True
+# Each testcase can take upto an hour or so depending on amount
+# of resources available on the system under test
+testcase: !mux
+    brk1:
+        name: brk1
+        postprocess: True
```

Comment:
remove  postprocess: True  in each variant and keep it above testcase: !mux  makes it available for each variant